bn-apps/memberships-management/membership-service-contracts-and-states/src/resources/migration/Schema.changelog-createTable.xml

```diff
@@ -0,0 +1,7 @@
+<changeSet author="R3 Corda" id="Create the table">
+    <createTable tableName="Schema">
+        <column name="member" type="NVARCHAR(255)"/>
+        <column name="BNO" type="NVARCHAR(255)"/>
+        <column name="status" type="NVARCHAR(255)"/>

```

Comment:
please make sure that name of the columns match the actual schema: https://github.com/corda/corda-solutions/blob/master/bn-apps/memberships-management/membership-service-contracts-and-states/src/main/kotlin/com/r3/businessnetworks/membership/states/Schemas.kt